File: DAY_0_FOUNDATION/Simple_Interest.py
principleAmount=int(input("enter the principle amount="))
rateOfInterest=float(input("Enter the rate of interest:"))
timePeriod=float(input("Enter the time period in years:"))
simpleinterest=(principleAmount*timePeriod*rateOfInterest)/100
totalAmount=simpleinterest+principleAmount
print("The simple interest is: ", simpleinterest)
print("The new total amount is: ", totalAmount)

File: Foundational/simpleinterest.cpp
//simple interest with total

#include <iostream>
using namespace std;

    int main()
    {
        double principal, rate, time, simpleInterest, totalAmount;

        cout << "Enter principal amount: ";
        cin >> principal;
        cout << "Enter rate of interest (per annum): ";
        cin >> rate;
        cout << "Enter time (in years): ";
        cin >> time;

        simpleInterest = (principal * rate * time) / 100;
        totalAmount = principal + simpleInterest;

        cout << "Simple Interest = " << simpleInterest << endl;
        cout << "Total Amount = " << totalAmount << endl;

        return 0;
    }